application/common/dos.py

```diff
@@ -25,13 +25,18 @@ class DoSService:
     name: str
     odscode: str
     address: str
+    town: str
     postcode: str
     web: str
     typeid: int
     statusid: int
     publicphone: str
     publicname: str
     servicename: str
+    easting: int
```

Comment:
Attributes easting and northing hold only numbers, not decimals?

Reply:
I've changed the type hints to floats to allow for decimal but as they are only type hints it shouldn't make a difference to the way the variables are used